tests/e2e/tidbcluster/serial.go

```diff
@@ -746,6 +753,39 @@ var _ = ginkgo.Describe("[tidb-operator][Serial]", func() {
 			pdClient, cancel, err := proxiedpdclient.NewProxiedPDClient(c, fw, ns, clusterName, false, nil)
 			framework.ExpectNoError(err, "create pdapi error")
 			defer cancel()
+
+			// wait for ReadyToScaleThresholdSeconds seconds
+			klog.Info("wait 300 seconds, auto scale should not happen with noise reduction")
+			time.Sleep(300 * time.Second)
```

Comment:
too much sleep time in these two e2e tests, it will take a long time to finish. can sleep time be eliminated? is it enough to cover this feature in unit tests?

Reply:
yeah, agree, one optimized way is to set ReadyToScaleThresholdSeconds to 30s when deploy operator, only only wait 30s here should already show noise reduction capability. what do you think?